client/main.js

```diff
@@ -15,35 +15,39 @@ function Event({ event }) {
   return <div className={ `event list-group-item ${ eventCategory ? eventCategory.toLowerCase() : null }` }>
     <h2>{ event.title } <EventTime time={event.time} /> </h2>
     <p>
-      <span>{ event.info && event.info.Lokacija ? event.info.Lokacija : null }</span>
-      <span> | </span>
-      <span>{ eventCategory }</span>
+      <h4>{ event.info && event.info.Lokacija ? event.info.Lokacija : null }</h4>
+      <span className="label label-primary">{ eventCategory }</span>
     </p>
     <p dangerouslySetInnerHTML={ eventDescription(event.description) }></p>
```

Comment:
Agreed! I think it should be part of event mapping, i.e. deserialization process after scraping? If you agree I plan to do some work on _backend_ side anyhow so I can do it in another PR alongside?